torch_xla/distributed/xla_dist.py

```diff
@@ -362,9 +362,11 @@ def _prepare_scripts(self, cmd):
       if self.tpuvm_mode:
         # Start the local tf server if it is not already running.
         script.append([
-            'python', '-m', self.XRT_RUN_SERVER_CMD,
+            'python', '-m', self.XRT_RUN_SERVER_CMD, '--port',
```

Comment:
Just noticing versioning issue here too.. We instead might want to make python3 default on TPU VM image to be less hacky.

Reply:
Alternatively you can assume we want same py version as initial runner so just using this would work:
https://github.com/pytorch/xla/blob/371bdf462a6b7576e294ae39e5ba23d8509d7834/contrib/scripts/env-setup.py#L49

Reply:
I think it is better to use `python3` everywhere, I will update